### PR TITLE
Support for fixed tick locations

### LINKED ISSUE #2548: Need more control over ticks and gridlines
I'm trying to reproduce a graph known in audiology as a tone audiogram. It is a graph of hearing sensitivity (in Decibel) vs frequency (Herz).

It has some peculiar requirements:
1. vertical (decibel) axis is inverted with specific major grid lines. 
2. horizontal (frequency) axis is in fact a log axis but its ticks are linear, and should only show the ticks/gridlines 125, 250, 500, 2000, 4000, 8000 (note the doubling of frequency values).
3. the aspect ratio should be such that a the area bounded by 20 dB and a frequency doubling (say 250-500, or 4000-8000 Hz) is square.

1 is not a problem in Bokeh. 3 could be achieved (more or less) by fiddling with the plot size. 
2 is the real challenge. Ideally I'd like to specify exactly which ticks and gridlines are drawn (eg by specifying a list of values).

formulier_-_toon-_en_spraakaudiogram_-_1_2_pdf__1_page_

### PR DESCRIPTION
**Pull request by bryevdv**
issues: closes #2548

### CODE REVIEW

**Comment by bryevdv** [comment]
@damianavila another small one to consider

**Comment by bryevdv** [comment]
screen shot 2015-07-18 at 4 37 53 pm

File: tests/travis/test.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 - path: "charts"
 - path: "plotting/file"
   type: file
-  skip: ["image.py", "image_rgba.py", "ajax_source.py"]
+  skip: ["image.py", "image_rgba.py", "ajax_source.py", "choropleth.py"]
```
Comment (damianavila) [inline tests/travis/test.yaml]: finally ;-)

**Comment by damianavila** [comment]
LGTM too..

**Pull request merged**